octoprint_discordremote/shared_vars.py

```diff
@@ -0,0 +1,8 @@
+#Used for accessing variables across classes when imports are not viable
```

Comment:
@OctopusProteins Im pretty sure there is not need for this. The plugin (__init__.py) class is passed into every sub class, so anything that needs to be centralised should go in the plugin, or use get_settings().

Reply:
On it.

Reply:
Fixed by ea06074